### PR TITLE
Add presentation-only VideoControls component

### PR DESCRIPTION
**Pull request by horatiualmasan**
## What has changed and why?

First of 4 PRs splitting #1649 (new player controls) into independently
reviewable pieces.

Adds a presentation-only `VideoControls` bar: a full-width scrubber plus
transport buttons (play/pause, mute, fullscreen). It owns no playback state
— it renders props and calls back on user intent — so it stays decoupled
from the player and its full-width scrubber can later host aligned timeline
overlays. Includes pure helpers (`formatTime`, `clampPercent`,
`timeFromClientX`) and a barrel export. Not wired into `VideoPlayer` yet

## How has it been tested?

New unit tests for the control bar (play/pause affordance, time display,
scrubber seek by pointer/keyboard, mute/fullscreen callbacks).

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [ ] Yes
- [x] Not needed (internal change)

## Summary by CodeRabbit

- **New Features**
  - Added video playback controls with play/pause, mute, fullscreen, time display, and seeking.
  - Supports pointer dragging and keyboard arrow-key navigation on the timeline.
  - Added accessible slider behavior and visual playback progress.

- **Documentation**
  - Added interactive and state-based Storybook examples for the video controls.

- **Tests**
  - Added coverage for playback, seeking, keyboard navigation, mute, fullscreen, and edge cases.

### CODE REVIEW

File: lightly_studio_view/src/lib/components/VideoControls/VideoControls.svelte [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        isMuted,
+        isFullscreen,
+        onPlayPause,
+        onSeek,
+        onToggleMute,
+        onToggleFullscreen,
+        class: className
+    }: VideoControlsProps = $props();
+
+    const SEEK_STEP_S = 5;
```
Comment (LeonardoRosaa) [inline lightly_studio_view/src/lib/components/VideoControls/VideoControls.svelte:37]: Should the step be proportional to the video’s duration? My suspicion is that a video shorter than 5 seconds will skip straight to the end when you press the right arrow

**Review by LeonardoRosaa — COMMENTED** [review]
Could you add a storybook?

File: lightly_studio_view/src/lib/components/VideoControls/VideoControls.svelte (hunk 1 above)
Comment (horatiualmasan) [reply lightly_studio_view/src/lib/components/VideoControls/VideoControls.svelte:37]: updated for short videos, for longer ones I'd leave 5s

**Review by LeonardoRosaa — APPROVED** [review]
